src/main/java/uk/gov/hmcts/reform/iacaseapi/infrastructure/controllers/PreSubmitCallbackController.java

```diff
@@ -54,13 +49,23 @@ public PreSubmitCallbackController(
             code = 200,
             message = "Transformed Asylum case data, with any identified error or warning messages",
             response = PreSubmitCallbackResponse.class
+        ),
+        @ApiResponse(
+            code = 400,
+            message = "Bad Request",
+            response = PreSubmitCallbackResponse.class
+        ),
+        @ApiResponse(
+            code = 500,
+            message = "Internal Server Error",
+            response = PreSubmitCallbackResponse.class
         )
     })
     @PostMapping(path = "/ccdAboutToStart")
     public ResponseEntity<PreSubmitCallbackResponse<AsylumCase>> ccdAboutToStart(
-        @ApiParam(value = "Asylum case data", required = true) @NotNull @RequestBody String source
+        @ApiParam(value = "Asylum case data", required = true) @NotNull @RequestBody Callback<AsylumCase> callback
```

Comment:
I think if this cannot be deserialized, it will return `HTTP 415` -- can we add that to the Api documentation, and perhaps it replaces the vanilla `400` code, not sure?